server/index.js

```diff
@@ -79,6 +111,104 @@ app.get('/:current', (req, res) => {
   res.sendFile(path.join(__dirname, '../dist/index.html'));
 });
 
+app.post('/comment', async (req, res) => {
+  try {
+    const lastComment = await db.lastComment();
+
+    const newComment = {
+      comment_id: lastComment[0].comment_id + 1,
+      user_id: req.body.user_id,
+      song_id: req.body.song_id,
+      content: req.body.content,
+      time_stamp: req.body.time_stamp,
+    };
+
+    const storedComment = await db.saveComment(newComment);
+
+    res.status(201).send({
+      success: true,
+      data: storedComment,
+    });
+  } catch (error) {
+    console.error(error);
+    res.status(400).json({
+      success: false,
+      msg: error,
+    });
+  }
+});
+
+app.post('/comment/:id', async (req, res) => {
+  try {
+    const { id } = req.params;
+
+    const newComment = {
+      comment_id: id,
+      user_id: req.body.user_id,
+      song_id: req.body.song_id,
+      content: req.body.content,
+      time_stamp: req.body.time_stamp,
+    };
+
+    const storedComment = await db.saveComment(newComment);
+
+    res.status(201).send({
+      success: true,
+      data: storedComment,
+    });
+  } catch (error) {
+    console.error(error);
+    res.status(400).json({
+      success: false,
+      msg: error,
+    });
+  }
+});
```

Comment:
The logic for these two routes appears to be largely the same. I'm not sure what's customary in terms of routes, but consider whether this can be simplified if done differently, e.g., with query parameters, so that they can be combined and the presence of an `id` can be checked conditionally.

Also, the way your tests are set up, your updates are being done on the `PUT` rather than the `POST`, see line 129 of test/test.js, so I'd expect to see the `id`-specific route that starts on line 141 here be an `app.put` rather than a `.post`. I don't think an `id`-specific  `app.post` is necessary.